packages/apollos-church-api/config.yml

```diff
@@ -32,7 +32,7 @@ ALGOLIA:
   APPLICATION_ID: ${ALGOLIA_APP_ID}
   API_KEY: ${ALGOLIA_API_KEY}
   # TODO: should we support multiple indices? how?
-  SEARCH_INDEX: test_ContentItems
+  SEARCH_INDEX: ${ALGOLIA_SEARCH_INDEX}
```

Comment:
did you add this to heroku for all environments?

Reply:
@redreceipt I did! Is `staging_ContentItems` right for `staging`?